babel/dates.py

```diff
@@ -1282,6 +1286,14 @@ def format_timezone(self, char, num):
                 return get_timezone_name(self.value.tzinfo, width,
                                          uncommon=True, locale=self.locale)
             return get_timezone_location(self.value.tzinfo, locale=self.locale)
+        # Included additional elif condition to add support for 'X' in timezone format
+        elif char in ['x','X']:
```

Comment:
Use `elif char in "xX"` or `elif char == "x" or char == "X"` instead of constructing a list here.